addons/hr_expense/migrations/9.0.2.0/post-migration.py

```diff
@@ -51,7 +51,7 @@ def hr_expense(cr):
                 create_date, write_date, create_uid, write_uid,
                 product_uom_id, unit_amount, quantity, product_id)
                 SELECT company_id, currency_id, journal_id, employee_id,
-                state, date, account_move_id, name, bank_journal_id, 0.00.
+                state, date, account_move_id, name, bank_journal_id, 0.00,
```

Comment:
this one fell on my feet too: https://github.com/OCA/OpenUpgrade/pull/603/files#diff-c8cd1d437f6fef2bc83ae6d87cd9f753R54